packages/grid/test/filtering.test.js

```diff
@@ -18,7 +18,7 @@ class FilterWrapper extends PolymerElement {
         }
       </style>
       <vaadin-grid-filter path="foo" value="[[_filterValue]]" id="filter">
-        <input slot="filter" value="{{_filterValue::input}}" />
+        <input value="{{_filterValue::input}}" />
```

Comment:
Wanna add an aria-label here if feasible? Like `Filter for {columnname}`?

Reply:
I don't think we need that in a test file.

Reply:
Sorry was more about the real Impl 🙂

Reply:
Yes, we should eventually add that. But let's consider doing that in a separate PR.
This needs to be also implemented for `vaadin-crud`, see https://github.com/vaadin/web-components/issues/3423